File: src/main/java/com/likelionknu/applyserver/admin/data/dto/response/AdminUserResponseDto.java
package com.likelionknu.applyserver.admin.data.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
@AllArgsConstructor
public class AdminUserResponseDto {

    @JsonProperty("user_id")
    private Long userId;

    private String name;
    private String email;
    private String role;
    private String depart;

    @JsonProperty("created_at")
    private LocalDateTime createdAt;

    @JsonProperty("last_accessed_at")
    private LocalDateTime lastAccessedAt;
}

File: src/main/kotlin/com/likelionknu/applyserver/admin/data/dto/response/AdminUserResponseDto.kt
package com.likelionknu.applyserver.admin.data.dto.response

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.LocalDateTime

data class AdminUserResponseDto(
    @field:JsonProperty("user_id")
    val userId: Long,

    val name: String,

    val email: String,

    val role: String,

    val depart: String?,

    @field:JsonProperty("created_at")
    val createdAt: LocalDateTime?,

    @field:JsonProperty("last_accessed_at")
    val lastAccessedAt: LocalDateTime?
)